### ISSUE TITLE
Can i use this with petite-vue?

### ISSUE THREAD

**Issue by ricky11** [ingram-projects/animxyz#77]
petite-vue does not have vue transitions built in? in this case can i use it in normal html mode?

**Comment by mattaningram** [comment]
I don't think you can import the Vue plugin/component, but you can use the AnimXYZ CSS file it as if it was normal HTML and just use the `xyz` attribute and classes.

**Comment by milesingrams** [comment]
Only way to use it without the core vue transitions would be to go the pure CSS route. Definitely effective but wont have exit animation capability. Still might look good though!
